src/drive/web/modules/search/components/BarSearchInputGroup.jsx

```diff
@@ -23,6 +31,13 @@ const BarSearchInputGroup = ({ children, isBusy, isFocus }) => {
           <Icon icon={Magnifier} color="#95999D" />
         )
       }
+      append={
+        isDirty ? (
```

Comment:
nit: `isDirty` looks a little confusing for me to inform about text in an input. I would prefer `isEmpty` or `isNotEmpty` or something like this.